packages/core/npm-aqua-compiler/.gitignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
Why not stroing `package-lock.json` in git?

Reply:
Not sure should i keep it or no.
When this file is present and some dependencies changes e.g. someone adds more packages to tests, npm just breaks without meaningful error.

Reply:
Without this file in git there is a less chance to fight with strange bugs when developing tests

Reply:
From the other side this file will keep the tests more persistent, would be easier to track changes. Probably i will keep it, thanks.